### ISSUE TITLE
508-defect-2 [SCREENREADER]: Headings should be consistent

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#25736]
### 1. `h2` "You have an upcoming appointment" should be a `h3` because it is a hierarchically below the step
<img width="1354" alt="Screen Shot 2021-06-03 at 1 40 52 PM" src="https://user-images.githubusercontent.com/14154792/120688898-849a8600-c471-11eb-90e4-ce5cefa082d9.png">

###  2. This _looks_ like a `h2` but is actually a `p`. I think this is a design and content problem that needs to be discussed. 
<img width="1418" alt="Screen Shot 2021-06-03 at 1 47 16 PM" src="https://user-images.githubusercontent.com/14154792/120689534-38037a80-c472-11eb-9d10-7ff5dcbd60fb.png">

#### Semantically, it would make more sense with cleaner content and as a `h3` that's _not_ contained within a `section`

<img width="1436" alt="Screen Shot 2021-06-03 at 1 50 06 PM" src="https://user-images.githubusercontent.com/14154792/120689912-9d576b80-c472-11eb-8801-c41430cc6b5b.png">
